src/FrontDoor/FrontDoor/help/Set-AzFrontDoor.md

```diff
@@ -130,6 +182,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -BackendPoolsSetting
```

Comment:
An example with `-BackendPoolsSettings` would be better

Reply:
Good point. I updated the existing examples to use -BackendPoolsSetting and added an example using -DisableCertificateNameCheck.